packages/volto/news/6354.documentation

```diff
@@ -0,0 +1 @@
+Added the new configuration for Markdown in the vscodeSettingsJSON object @aadityaforwork
```

Comment:
Change log notes should describe what the change actually does, and be from the perspective of the developer or user who reads it. Implementation details cloud the purpose.

```suggestion
Added the configuration for VSCode not to reformat Markdown and MyST files. @aadityaforwork
```